x-pack/plugins/lens/public/xy_visualization/xy_config_panel.tsx

```diff
@@ -68,3 +85,73 @@ export function LayerContextMenu(props: VisualizationLayerWidgetProps<State>) {
     </EuiFormRow>
   );
 }
+
+export function XyToolbar(props: VisualizationToolbarProps<State>) {
+  const [open, setOpen] = useState(false);
+  const hasNonBarSeries = props.state?.layers.some(
+    (layer) => layer.seriesType === 'line' || layer.seriesType === 'area'
+  );
+  return (
+    <EuiFlexGroup justifyContent="flexEnd">
+      <EuiFlexItem grow={false}>
+        <EuiPopover
+          panelStyle={{ width: 400 }}
```

Comment:
nit: my preference would be to remove inline styles and use panelClassName instead, but if you prefer to leave it, you could remove line 99 (`panelClassName=''`)

Reply:
Looks like the `.lnsXyToolbar__popover` class and style were created but not applied to the component.

Reply:
I wanted to do this and prepared a stylesheet file: https://github.com/elastic/kibana/pull/69820/files#diff-00b8d94b8fe588750ac3944e58206295

Just forgot to link the class here. It's fixed now.